internal/webhookcert/resources_test.go

```diff
@@ -0,0 +1,37 @@
+package webhookcert
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+)
+
+func TestApplyWebhookConfigResources(t *testing.T) {
+	scheme := runtime.NewScheme()
+	ctx := context.Background()
+	caBundle := []byte("test-ca-bundle")
+	config := Config{
+		ServiceName: types.NamespacedName{
+			Name:      "test-service",
+			Namespace: "test-namespace",
+		},
+		ValidatingWebhookName: types.NamespacedName{
+			Name: "test-validating-webhook",
+		},
+		MutatingWebhookName: types.NamespacedName{
+			Name:      "test-mutating-webhook",
+			Namespace: "test-namespace",
+		},
+	}
+
+	client := fake.NewClientBuilder().WithScheme(scheme).Build()
+
+	t.Run("should fail apply webhook config resources", func(t *testing.T) {
```

Comment:
what is the purpose of this test? to check that `applyWebhookConfigResources()` will fail if the initial validating webhooks and mutating webhooks don't exist? I think we can drop this test

Reply:
Also, it is a better practice to test the logic through the exported functions from the pkg. In this case, it would be the `EnsureCertificate` func

Reply:
This test not more required and removed